### DISCUSSION TITLE
Support to validate my thoughts concerning simplified pyrolysis scenario in a warehouse

### DISCUSSION THREAD

**Discussion by IngSillo** [firemodels/fds#12830] [Q&A]
Dear all,

I kindly ask for your expert opinion concerning a simplified modelling scenario I'm developing to learn FDS. I've already designed a satisfactory model geometry mesh.

The model aims to simulate a _worst case scenario_ of a logistic warehouse where item are stored on wooden pallets. I must obtain: 

1. the temperature increase on wrought-iron pillars;
2. temperature increase on the besides wooden pallet to understand when will they start to burn.

The local fire brigate suggested us to model using the below table in order to have a cautelative HRR Curve.

Parameter | Value for Other Activities
--- | ---
Fire growth rate "t² fire" | 75 s (ultra-fast)
Total HRR | 50 MW
HRR per m² of fire surface | 500 -1000 kW/m² <sup>[1]</sup>
Soot yield Y_soot (Pre/Post-flashover) | 0.18 kg/kg <sup>[4]</sup> / 0.36 kg/kg <sup>[4]</sup>
Carbon monoxide yield Y_CO (Pre/Post-flashover) | 0.10 kg/kg <sup>[5]</sup> / 0.40 kg/kg <sup>[5]</sup>
Effective heat of combustion ΔHC | 20 MJ/kg <sup>[3]</sup>
Carbon dioxide yield Y_CO2 | 1.5 kg/kg <sup>[3,6]</sup>
Water yield Y_H2O | 0.82 kg/kg <sup>[3,6]</sup>
Radiative fraction | 35% <sup>[3]</sup>

References:
[1] Consider in alternative to total HRR, considering the maximum fire surface area, similar to a compartment fire in the case of uniformly distributed fire load, which can be a localized fire scenario.
[2] Robbins A P, wade C A, Study Report No.185 “Soot Yield Values for Modelling Purposes – Residential Occupancies”, BRANZ, 2008.
[3] "C/VM2 Verification Method: Framework for fire safety design", New Zealand Building Code
[4] "SFPE handbook of fire protection engineering", NFPA, 4th ed., 2008. Tables 3.4-16, page 3-142, from polyurethane flexible foams.
[5] Stec A A, Hull T R, "Fire Toxicity", Woodhead Pub., 2010. § 2.4 cont on page 1.25 (underventilated fire)
[6] Alternatively to the yields Y_CO2 and Y_H2O, the calculation code for the generic compound CH2O0.5 can be imported.

I'm confident I can handle the modeling of a the above using a dedicated excel spreadsheet to normalize the HRR and eventually the use of Control Devices to model the _sprinkler effect_ governing a steady state fire stopping the growth of the HRR at a given time.

Now might you please check my assumption reported below?

- Given the request of the local fire brigade it is legit to simply define that any materials will burn according to the above represented HRR curve and related RAMP even if this is not true in real life.

- To centre the target of the model (1,2 above) I must carefully design for &MATL properties in order to have an effective representation of the heat trasport phenomena (dominated by conducion). This won't interfere anyhow with the exaggerated representation of the HRR Curve of above. 

- Once the materials are well represented, I will be able to measure the Temperature on wrought-iron pillars in time and on the beside pallets.

- The besides pallets will take fire if only an IGNITION TEMPERATURE is defined through FDS &MATL or &SURF namelists. There are no other way around to do so with FDS (simplified pyrolisis).

At last, two questions:

1. which &REACT must I assume? Given the fact that the HRR curve is given I thought that any predefined REACT wuold fit the case.
2. Is radiative fraction key? which is the factor that I must set to govern the HRR with 35% of radiative fraction?

**Comment by drjfloyd** [comment]
1. The fire brigade has requested specific yields of C, CO, CO2, and H2O. It is not true that any fuel will give you those values. For example, if you chose SOOT as your fuel species, then there would be no H2O made. You will need to define some physically plausible fuel species that when it burns will give the requested yields.

2. Please read  the User's Guide on how to set the radiative fraction.

**Reply by IngSillo** [reply]
I went through the guide again.

I apologize for question two on the radiative issue. 35% should be the standard value by the way. I just had forgotten. Thanks for the reply.

Also, I agree with the first point you mentioned.

I will simply model a realistic fire for pallets that has wooden stuff to burn.

Hence: 

&REAC FUEL ='GENERIC_FUEL'
              SOOT_YIELD=0.18
              CO_YIELD=0.10
              C=1
              H=2
              O= 0.5
HEAT OF COMBUSTION=20000/

The local brigade indicates yields for CO and SOOT after the flashover which almost double the pre-flashover values. 

Given that I will "cut" the HRR curve thanks to the deployment of sprinkler systems, is it legit to simply use only the pre flashover yields? 

Shall I design a dedicated RAMP for the yields otherwise? If it is physically legit?

**Comment by drjfloyd** [comment]
Whether or not pre or post flashover yields are appropriate is a decision you need to make and jusify based on your specific usage of FDS and your knowledge of fire dynamics.

You cannot RAMP the yield inputs.
